config/initializers/analytics_ruby.rb

```diff
@@ -17,7 +17,7 @@ def initialize(status, error_message)
 
   SEGMENT_CLIENT = Segment::Analytics.new(
     {
-      write_key: ENV['SEGMENT_WRITE_KEY'],
+      write_key: ENV.fetch('SEGMENT_WRITE_KEY'),
```

Comment:
We do not want SEGMENT_WRITE_KEY to be a requirement for the application. It used by the lago team to monitor the usage of the tool. It should not be required to perform the specs

I would recommend to use a fallback value instead:

```suggestion
      write_key: ENV.fetch('SEGMENT_WRITE_KEY', 'change_me'),
```

Reply:
Done, I used `changeme` to be consistent with the rest of the app:

```
.env.dist:DATABASE_URL=postgresql://lago:changeme@localhost:5432/lago_test
config/database.yml:    password: changeme
config/database.yml:    password: changeme
config/initializers/analytics_ruby.rb:      write_key: ENV.fetch('SEGMENT_WRITE_KEY', 'changeme'),
```